scenarioo-client/test/protractorE2E/specs/show_step.js

```diff
@@ -146,10 +150,101 @@ useCase('Show step')
         scenario('Step without HTML source attached')
             .description('If the step data contains no html source data, the HTML tab should not be displayed at all')
             .it(function () {
-
                 stepPage.goToPage('/step/Donate/find_donate_page/startSearch.jsp/0/0');
                 stepPage.assertHtmlTabIsHidden();
                 step('A step with no HTML source attached');
             });
 
+
+        scenario('ScenarioPage with comparisons')
```

Comment:
Thanks a lot for improving on that. Yes, true, we have to overwork our test suite structure once.
But it has become allready a lot better. Thanks.